src/tests/bitsets/test_stdlib_bitset_64.f90

```diff
@@ -0,0 +1,744 @@
+program test_stdlib_bitset_64
+    use, intrinsic :: iso_fortran_env, only : int8, int16, int32, int64
+    use stdlib_bitsets
+    character(*), parameter :: &
+        bitstring_0   = '000000000000000000000000000000000', &
+        bitstring_33  = '100000000000000000000000000000000', &
+        bitstring_all = '111111111111111111111111111111111'
+    type(bitset_64) :: set0, set1, set2, set3, set4, set5
+    integer         :: status
+    character(:), allocatable :: string0
+
+    call test_string_operations()
+
+    call test_io()
+
+    call test_initialization()
+
+    call test_bitset_inquiry()
+
+    call test_bit_operations()
+
+    call test_bitset_comparisons()
+
+    call test_bitset_operations()
+
+contains
+
+    subroutine test_string_operations()
+        character(*), parameter:: procedure = 'TEST_STRING_OPERATIONS'
+
+        write(*,*)
+        write(*,*) 'Test string operations: from_string, read_bitset, ' // &
+            'to_string, and write_bitset'
+
+        call set0 % from_string( bitstring_0 )
+        if ( bits(set0) /= 33 ) then
+            error stop procedure // ' from_string failed to interpret ' // &
+                'bitstring_0 size properly.'
+        else if ( .not. set0 % none() ) then
+            error stop procedure // ' failed to interpret bitstring_0 ' // &
+                'value properly.'
+        else if ( set0 % any() ) then
+            error stop procedure // ' failed to interpret bitstring_0 ' // &
+                'value properly.'
+        else
+            write(*,*) 'from_string transferred bitstring_0 properly into set0'
+        end if
+
+        call set1 % from_string( bitstring_all )
+        if ( bits(set1) /= 33 ) then
+            error stop procedure // ' from_string failed to interpret ' // &
+                'bitstring_all size properly.'
+        else if ( set1 % none() ) then
+            error stop procedure // ' failed to interpret bitstring_all ' // &
+                'value properly.'
+        else if ( .not. set1 % any() ) then
+            error stop procedure // ' failed to interpret bitstring_all ' // &
+                'value properly.'
+        else if ( .not. set1 % all() ) then
+            error stop procedure // ' failed to interpret bitstring_all ' // &
+                'value properly.'
+        else
+            write(*,*) 'from_string transferred bitstring_all properly ' // &
+                'into set1'
+        end if
+
+        call set3 % read_bitset( bitstring_0, status )
+        if ( status /= success ) then
+            write(*,*) 'read_bitset_string failed with bitstring_0 as expected.'
```

Comment:
Is an `error stop` not missing here?

Reply:
I guess an error stop if the expected failure does not occur is appropriate.